From 49dc2a687175f9671a159df38971a15287dae18c Mon Sep 17 00:00:00 2001
From: Chris Leech <cleech@redhat.com>
Date: Mon, 24 Feb 2014 09:33:33 -0800
Subject: [PATCH] Revert "iscsiadm: return error when login fails"
 
This reverts commit fc2a8e9a2911bc76f961fe3e4a159fab9b8b9691.
 
Done to address RHBZ #1015563
---
 usr/session_mgmt.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)
 
diff --git a/usr/session_mgmt.c b/usr/session_mgmt.c
index 0500f15..1e1f2bc 100644
--- a/usr/session_mgmt.c
+++ b/usr/session_mgmt.c
@@ -178,12 +178,12 @@ int iscsi_login_portal(void *data, struct list_head *list, struct node_rec *rec)
 		goto done;
 	}
 	if (session_count >= rec->session.nr_sessions) {
-		log_warning("%s: %d session%s requested, but %d "
+		log_debug(1, "%s: %d session%s requested, but %d "
 			  "already present.",
 			  rec->iface.name, rec->session.nr_sessions,
 			  rec->session.nr_sessions == 1 ? "" : "s",
 			  session_count);
-		rc = ISCSI_ERR_SESS_EXISTS;
+		rc = 0;
 		goto done;
 	}
 
-- 
2.26.2
 